rpc/handler.go

```diff
@@ -388,7 +387,7 @@ func (h *handler) handleSubscribe(cp *callProc, msg *jsonrpcMessage) *jsonrpcMes
 func (h *handler) runMethod(ctx context.Context, msg *jsonrpcMessage, callb *callback, args []reflect.Value) *jsonrpcMessage {
 	result, err := callb.call(ctx, msg.Method, args)
 	if err != nil {
-		return msg.errorResponse(err)
+		return msg.errorResponse(&internalServerError{cause: err})
```

Comment:
This change is not correct. Here, `err` is the error value returned by the RPC method implementation. When the method returns an error that implements `rpc.Error` or `rpc.DataError`, we want to return the code and data contained in `err`.

However, with this change, the error code is overwritten.

Reply:
what if the RPC method returns non `rpc.Error`, should we wrap it with internalServerError?